src/runtime/components/AlgoliaDocSearch.vue

```diff
@@ -0,0 +1,180 @@
+<template>
+  <div id="docsearch">
+    <button type="button" class="DocSearch DocSearch-Button" aria-label="Search" />
+  </div>
+</template>
+
+<script setup lang="ts">
+import type { PropType } from 'vue'
+import { withoutTrailingSlash } from 'ufo'
+import { useRuntimeConfig } from 'nuxt3'
+import type { DocSearchOptions } from '../../types'
+// @ts-ignore - These are Nuxt3 aliases
+import { useRoute, useRouter, onMounted, watch, computed } from '#imports'
```

Comment:
Having both nuxt3 and #imports can lead to import issues. Could you please merge them into one to avoid that?